examples/preact/basic-dynamic-columns/src/index.css

```
@@ -0,0 +1,365 @@
+.pin-button,
+.compact-input,
+.filter-input,
+.filter-select,
+.page-size-input,
+.text-input,
+.number-input,
+.wide-action-button,
+.primary-action,
+.secondary-action,
+.success-action {
+  border: 1px solid currentColor;
```

Stylelint (17.14.0)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)